common/element_handle.go

```diff
@@ -238,8 +238,12 @@ func (h *ElementHandle) checkHitTargetAt(apiCtx context.Context, p Position) (bo
 			return injected.checkHitTargetAt(node, point);
 		}
 	`)
+	opts := evaluateOptions{
+		forceCallable: true,
+		returnByValue: true,
+	}
```

Comment:
😄 Not here because config structs that are not expected to grow or don't need to be passed as a `nil` value are good. Having them optional here won't do any good because the purpose was to make things more straightforward and reduce possible bugs in the future. Accepting successive primitive values is not a good pattern.